### PR TITLE
Hotfix/hide get api key

### LINKED ISSUE #2573: Hide "Get API Key" button when EMQ is selected as a proxy for API

### PR DESCRIPTION
**Pull request by frenchbread**
Closes #2573 

### Changes
- Fixed indentation
- Added template helper
- Added check to template

### CODE REVIEW

**Comment by frenchbread** [comment]
Please review 👀

File: api_keys/client/api_key.js [hunk 1]
```diff
@@ -107,4 +107,10 @@ Template.apiKey.helpers({
 
     return apiKey;
   },
+  showButton () {
+    // Get proxyBackend from template data
+    const proxyBackend = Template.currentData().proxyBackend;
+
+    return proxyBackend.type === 'apiUmbrella';
```
Comment (brylie) [inline api_keys/client/api_key.js]: What happens if `proxyBackend` is `undefined`?

Comment (frenchbread) [inline api_keys/client/api_key.js]: Currently both proxies are set in a way to contain either "apiUmbrella" or "emq" proxy backend type. And `type` is required field.

Comment (brylie) [inline api_keys/client/api_key.js]: Would it be possible for `proxyBackend` to be `undefined`?

Comment (frenchbread) [inline api_keys/client/api_key.js]: I can't think of any scenario when `proxyBackend` is `undefined`

Comment (brylie) [inline api_keys/client/api_key.js]: Alright, we just get a bug about 'cannot access *property* of *undefined*' about once per sprint.

Comment (brylie) [inline api_keys/client/api_key.js]: So, just make sure to check for object before using a property. For safety.

Comment (brylie) [inline api_keys/client/api_key.js]: You can, for example, use the Meteor check() package, or just an `if (thing && thing.property) {// access property }`

Comment (frenchbread) [inline api_keys/client/api_key.js]: Ok, on it.

Comment (frenchbread) [inline api_keys/client/api_key.js]: `undefined` might have occurred when API is added and `proxyBackend` has not been configured yet. I've added check that hides the button if no proxy is attached to API.

**Comment by brylie** [comment]
OK, please resolve conflicts.

**Comment by frenchbread** [comment]
Done 👍

**Pull request merged**